cypress-tests/cypress/e2e/editor/widget/textInputHappyPath.cy.js

```diff
@@ -0,0 +1,217 @@
+import { fake } from "Fixtures/fake";
+import { textInputText } from "Texts/textInput";
+import { commonWidgetText } from "Texts/common";
+import { commonSelectors, commonWidgetSelector } from "Selectors/common";
+import { buttonText } from "Texts/button";
+import { verifyControlComponentAction } from "Support/utils/textInput";
+import {
+  openAccordion,
+  verifyAndModifyParameter,
+  openEditorSidebar,
+  verifyAndModifyToggleFx,
+  addDefaultEventHandler,
+  verifyComponentValueFromInspector,
+  selectColourFromColourPicker,
+  verifyBoxShadowCss,
+  verifyLayout,
+  verifyTooltip,
+  editAndVerifyWidgetName,
+  verifyPropertiesGeneralAccordion,
+  verifyStylesGeneralAccordion
+} from "Support/utils/commonWidget";
+
+describe('Text Input', ()=>{
+
+  beforeEach(()=>{
+    cy.appUILogin();
+    cy.createApp();
+    cy.dragAndDropWidget('Text Input');
+  });
+
+  it("should verify the properties of the text input widget", ()=>{
+    const data = {};
+    data.appName = `${fake.companyName}-App`;
+    data.widgetName = fake.widgetName;
+    data.customText = fake.firstName;
+    data.alertMessage = fake.randomSentence;
+    data.tooltipText = fake.randomSentence;
+
+    cy.renameApp(data.appName);
+
+    openEditorSidebar(textInputText.defaultWidgetName);
+    editAndVerifyWidgetName(data.widgetName)
+    openAccordion(commonWidgetText.accordionProperties);
+    verifyAndModifyParameter(commonWidgetText.labelDefaultValue, data.customText);
+    cy.forceClickOnCanvas();
+    cy.get(commonWidgetSelector.draggableWidget(data.widgetName)).verifyVisibleElement("have.value", data.customText);
+
+    verifyComponentValueFromInspector(data.widgetName, data.customText)
+    cy.forceClickOnCanvas();
+    cy.get(commonWidgetSelector.draggableWidget(data.widgetName)).clear()
+
+    data.customText = fake.randomSentence;
+    openEditorSidebar(data.widgetName);
+    openAccordion(commonWidgetText.accordionProperties);
+    verifyAndModifyParameter(commonWidgetText.labelPlaceHolder, data.customText);
+    cy.forceClickOnCanvas();
+    cy.get(commonWidgetSelector.draggableWidget(data.widgetName))
+    .invoke('attr', 'placeholder').
+    should('contain', data.customText)
+
+    openEditorSidebar(data.widgetName);
+    openAccordion(commonWidgetText.accordionEvents);
+    addDefaultEventHandler(data.alertMessage);
```

Comment:
I think for widgets with exposed values, We can use the alert message with the value.
Please share your thoughts.

Reply:
Yeah, we can use exposed values on alert messages, CSA and custom messages. Updated on the latest commit